lib/cutest.rb

```diff
@@ -171,6 +171,14 @@ def assert_raise(expected = Exception)
       success
     end
   end
+  
+  # Assert that the block raises no exceptions.
+  def assert_nothing_raised
+    yield
+    success
+  rescue Exception => exception
```

Comment:
This also captures _everything_ included interrupt (^c), out of memory, stack overflow, etc.

Depending on cutest's assertion model, it also captures failed assertions. I just verified that this is true as well.